recipes/rapidyaml/all/conanfile.py

```diff
@@ -21,12 +21,16 @@ class RapidYAMLConan(ConanFile):
         "fPIC": [True, False],
         "with_default_callbacks": [True, False],
         "with_tab_tokens": [True, False],
+        "with_default_callback_uses_exceptions": [True, False],
+        "with_assert": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "with_default_callbacks": True,
         "with_tab_tokens": False,
+        "with_default_callback_uses_exceptions": False,
```

Comment:
Afaik this one should only be valid if `with_default_callbacks` is `true` as per https://github.com/biojppm/rapidyaml/compare/v0.5.0...v0.6.0#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR17-R19 - we might need to delete the option in `configure` if that is true